pyrad/tools.py

```diff
@@ -158,15 +157,12 @@ def EncodeDate(num):
     return struct.pack('!I', num)
 
 
-def DecodeString(str):
```

Comment:
Did this function return different data types in success and unsuccess path ? I don't known what Author meant...